Rules/TDM/Scripts/TDM.as

```diff
@@ -320,8 +320,8 @@ shared class TDMCore : RulesCore
 		{
 			rules.SetCurrentState(GAME);
 		}
-		else if (ticksToStart > 0 && rules.isWarmup()) //is the start of the game, spawn everyone + give mats
-		{
+		else if (ticksToStart > 29 && rules.isWarmup()) // is the start of the game, spawn everyone + give mats. 
```

Comment:
I believe this line is what's causing the error. Sudden death text comes from `kills_to_win`.